scripts/compile_bundle.sh

```diff
@@ -1,5 +1,6 @@
-#!/bin/bash
-set -x
+#!/usr/bin/env bash
+set -veo pipefail
+# can not use -u, because external env/vars.sh source'ing
```

Comment:
would you elaborate more on using `/usr/bin/env bash` and `set -veo pipefail`?

Reply:
These are common best practice for bash scripts. -e and -o pipefail stop the script on any errors. `set -u` is highly recommended, to fail on any undefined variables. This avoids undefined and unexpected behaviors, especially here were ENV vars change the build options. `/usr/bin//env bash` ensure a working script on OS's where bash is not under /bin/bash

i might add a change to enable / disable set -u / set +u only around the source .../env/vars.sh includes 

For CI/CD it would actually be wise to use a POSIX shell script, not bash, because the smaller Alpine docker images don't even have bash. However, given this script depends on oneAPI anyway and it's supported distribution, bash should be fine.